### PR TITLE
Add ET to prod rbac

### PR DESCRIPTION
**Pull request by jthmcts**
### Change description ###
Employment Tribunals Release 1.0
Add ET to prod role binding file

CR: CHG5008762

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml [hunk 1]
```diff
@@ -557,3 +557,19 @@ subjects:
     kind: ServiceAccount
 
 ---
+
```
Comment (adusumillipraveen) [inline k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml:560]: This is for flux v1 and shouldn't be needed for a new app.

Comment (petermoores-version1) [reply k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml:560]: @adusumillipraveen - Are you saying we don't need to add any RBAC role binding for Flux V2? In other words abandon this PR? Thanks

Comment (petermoores-version1) [reply k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml:560]: @adusumillipraveen - Hi, did you see my reply to your comment?

Comment (adusumillipraveen) [reply k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml:560]: yes pls, this is only for v1.

**Comment by petermoores-version1** [comment]
Not required for Flux V2, only Flux V1 and Employment Tribs is only going live with a new Reform ECM product.

**Pull request closed without merging**